README.md

```diff
@@ -1,2 +1,48 @@
-# BOCS Control
-Control scripts for the BOCS.
+# BOCS Data Collection
+
+Contains the source code for the `python3-bocs` Debian package that runs the BOCS air quality instrument, recording data to local storage.
+Released Deb files are available at <TODO>.
+
+# Setup Pi
+
+The RaspberryPi needs setting up before the BOCS software can be installed.
+
+  1. Install the latest version of [RaspberryPi OS with desktop](https://www.raspberrypi.com/software/) to a microSD card
+  2. Boot up the Pi (outside of the BOCS) and follow the onscreen instructions to setup locale, password, WiFi access (if needed), and **make sure to update the software**
+  3. Reboot
+  4. Run `sudo raspi-config`, go into Interface Options and enable both `SSH` and `Legacy Camera`
```

Comment:
Is SSH optional for those who want to connect to the instrument via the network?

Why do we enable `Legacy Camera`?

Reply:
> Is SSH optional for those who want to connect to the instrument via the network?
Yes

> Why do we enable Legacy Camera?
So the display works. When Raspbian switched to tracking Debian Bullseye they changed the video drivers [see here](https://www.raspberrypi.com/news/bullseye-camera-system/)